kpet/utils.py

```diff
@@ -19,17 +19,17 @@ class TemplateNotFound(Exception):
     """Raised when an template is not found"""
 
 
-def get_template_path(filename):
+def get_template_path(filename, dbdir=None):
```

Comment:
Just a thought: could this function add the `.xml` extension as well?

Reply:
But the arguments says `filename`, it should say filename_without_extension hehe.

Reply:
Yeah, I'm just saying it could be called `tree` instead, for example :)